interface/globals.php

```diff
@@ -4,6 +4,7 @@
 require_once(dirname(__FILE__) . "/../common/compatibility/Checker.php");
 
 use OpenEMR\Checker;
+use OpenEMR\Core\Header;
```

Comment:
I am guessing in the future the plan is to migrate twig out of global.php into it's own place that is only called when using a twig template. If this is the case, just rec. placing a comment with this use OpenEMR\Core\Header (or could break the PSR rules and bundle it with the twig stuff below) so we remember to remove it when that happens.